mne/_fiff/reference.py

```diff
@@ -430,7 +468,10 @@ def set_eeg_reference(
             "reference."
         )
 
-    return _apply_reference(inst, ref_channels, ch_sel, forward, ch_type=ch_type)
+    if ref_channels is dict:
```

Comment:
i don't think you'll want to test for identity here, but use isinstance instead

Reply:
Thanks so much for pointing out the amateur mistake!